lib/matplotlib/path.py

```diff
@@ -282,11 +283,26 @@ def __deepcopy__(self, memo=None):
         readonly, even if the source `Path` is.
         """
         # Deepcopying arrays (vertices, codes) strips the writeable=False flag.
-        p = copy.deepcopy(super(), memo)
+        if sys.version_info >= (3, 14):
+            from copy import _reconstruct, _keep_alive
+            rv = super().__reduce_ex__(4)
+            assert memo is not None
+            p = _reconstruct(self, memo, *rv)
+            if memo is not None:
+                memo[id(self)] = p
+                _keep_alive(self, memo)
+
+        else:
+            p = copy.deepcopy(super(), memo)
         p._readonly = False
         return p
 
-    deepcopy = __deepcopy__
+    def deepcopy(self, memo=None):
+        """
+        Return a shallow copy of the `Path`, which will share the
```

Comment:
I think this should be "deep copy" ?

Reply:
yes it should!  This is a copy-paste error in the copy code 🤣